hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -1,185 +1,310 @@
+
+Run this command and create file in the working directory.
+
+	cat > simulator.properties
+	
+Copy these lines into property file and fill with your properties.
 
 ```
-CLOUD_IDENTITY=~/ec2.identity
-CLOUD_CREDENTIAL=~/ec2.credential
+CLOUD_PROVIDER=aws-ec2
+CLOUD_IDENTITY=
+CLOUD_CREDENTIAL=
+MACHINE_SPEC=hardwareId=
+JDK_FLAVOR=
+JDK_VERSION=
 ```
 
-Create two text files in your home folder. The file `ec2.identity` should contain your access key and the file 
-`ec2.credential` should contain your secret key. 
+***NOTE:*** *For a full description of the file `simulator.properties`, please see the [Simulator.Properties File Description section](#simulator-properties-file-description). Also you can find sample simulator properties in the `dist/simulator-tests/simulator.properties`. You can copy the this file to working directory ,then you can edit according to your needs.*
+
+<br></br>
+***RELATED INFORMATION***
+
+*Please see the [Provisioner section](#provisioner) and the [Coordinator section](#coordinator) for the more `provisioner` and `coordinator` commands.*
+<br></br>
 
-***NOTE:*** *For a full description of the file `simulator.properties`, please see the [Simulator.Properties File Description section](#simulator-properties-file-description).*
+### Running the Test
 
-### Editing the test.properties file
+When you are in your working folder, execute the following commands step by step to run the test.
 
-You need to give the classpath of `Example` test in the file `test.properties` as shown below.
+* First of all, you need agents to run the test on them.
 
 
-```
-class=yourgroupid.ExampleTest
-maxKeys=5000
-putProb=0.4
-```
+	```provisioner --scale 4```
+	
 
-The property `class` defines the actual test case and the rest are the properties you want to bind in your test. If a
-property is not defined in this file, the default value of the property given in your test code is used. Please see the `properties` comment in the example code above.
+This line of command starts 4 EC2 instances, install Java and the agents.
 
-You can also define multiple tests in the file `test.properties` as shown below.
+The output of the command looks like the following.
 
 ```
-foo.class=yourgroupid.ExampleTest
-foo.maxKeys=5000
+INFO  09:05:06 Hazelcast Simulator Provisioner
+INFO  09:05:06 Version: 0.5, Commit: c6e82c5, Build Time: 18.06.2015 @ 11:58:06 UTC
+INFO  09:05:06 SIMULATOR_HOME: /disk1/hasan/hazelcast-simulator-0.5
+INFO  09:05:07 Loading simulator.properties: /disk1/hasan/exampleSandbox/simulator.properties
+INFO  09:05:07 ==============================================================
+INFO  09:05:07 Provisioning 4 aws-ec2 machines
+INFO  09:05:07 ==============================================================
+INFO  09:05:07 Current number of machines: 0
+INFO  09:05:07 Desired number of machines: 4
+INFO  09:05:07 GroupName: simulator-agent-hasan
```

Comment:
You have not configured groupname 'similator-agent-hasan' in your simulator.properties. Where does this come from???